changelog/Microsoft.DirectoryServices.json

```
@@ -110,6 +110,392 @@
+          "Target": "filterByCurrentUser"
+        },
+        {
+          "Id": "b3431682-f63b-46c0-9317-deb79df71eed",
+          "ApiChange": "EnumType",
+          "ChangedApiName": "connectedOrganizationState",
+          "ChangeType": "Addition",
+          "Description": "Added the **connectedOrganizationState** enumeration type.",
+          "Target": "connectedOrganizationState"
+        },
+        {
+          "Id": "b3431682-f63b-46c0-9317-deb79df71eed",
+          "ApiChange": "Resource",
+          "ChangedApiName": "azureActiveDirectoryTenant",
+          "ChangeType": "Addition",
+          "Description": "Added the [azureActiveDirectoryTenant](https://docs.microsoft.com/graph/api/resources/azureActiveDirectoryTenant?view=graph-rest-1.0) resource type.",
+          "Target": "azureActiveDirectoryTenant"
+        },
+        {
+          "Id": "b3431682-f63b-46c0-9317-deb79df71eed",
+          "ApiChange": "Resource",
+          "ChangedApiName": "domainIdentitySource",
+          "ChangeType": "Addition",
+          "Description": "Added the [domainIdentitySource](https://docs.microsoft.com/graph/api/resources/domainIdentitySource?view=graph-rest-1.0) resource type.",
+          "Target": "domainIdentitySource"
+        },
+        {
+          "Id": "b3431682-f63b-46c0-9317-deb79df71eed",
+          "ApiChange": "Resource",
+          "ChangedApiName": "externalDomainFederation",
+          "ChangeType": "Addition",
+          "Description": "Added the [externalDomainFederation](https://docs.microsoft.com/graph/api/resources/externalDomainFederation?view=graph-rest-1.0) resource type.",
+          "Target": "externalDomainFederation"
+        },
+        {
+          "Id": "b3431682-f63b-46c0-9317-deb79df71eed",
+          "ApiChange": "Resource",
+          "ChangedApiName": "identitySource",
+          "ChangeType": "Addition",
+          "Description": "Added the [identitySource](https://docs.microsoft.com/graph/api/resources/identitySource?view=graph-rest-1.0) resource type.",
+          "Target": "identitySource"
+        },
+        {
+          "Id": "b3431682-f63b-46c0-9317-deb79df71eed",
+          "ApiChange": "Resource",
+          "ChangedApiName": "connectedOrganization",
+          "ChangeType": "Addition",
+          "Description": "Added the [connectedOrganization](https://docs.microsoft.com/graph/api/resources/connectedOrganization?view=graph-rest-1.0) resource",
+          "Target": "connectedOrganization"
+        },
+        {
+          "Id": "b3431682-f63b-46c0-9317-deb79df71eed",
+          "ApiChange": "Resource",
+          "ChangedApiName": "entitlementManagement",
+          "ChangeType": "Addition",
+          "Description": "Added [entitlementManagement](https://docs.microsoft.com/graph/api/resources/entitlementmanagement-root?view=graph-rest-1.0) resources",
+          "Target": "entitlementManagement"
+        }
```

Global: Just confirm the punctuation is there for all objects.
```suggestion
        {
          "Id": "b3431682-f63b-46c0-9317-deb79df71eed",
          "ApiChange": "Resource",
          "ChangedApiName": "connectedOrganization",
          "ChangeType": "Addition",
          "Description": "Added the [connectedOrganization](https://docs.microsoft.com/graph/api/resources/connectedOrganization?view=graph-rest-1.0) resource.",
          "Target": "connectedOrganization"
        },
        {
          "Id": "b3431682-f63b-46c0-9317-deb79df71eed",
          "ApiChange": "Resource",
          "ChangedApiName": "entitlementManagement",
          "ChangeType": "Addition",
          "Description": "Added [entitlementManagement](https://docs.microsoft.com/graph/api/resources/entitlementmanagement-root?view=graph-rest-1.0) resource.",
          "Target": "entitlementManagement"
        }
```